src/middleware/checkOwner.js

```diff
@@ -31,5 +31,19 @@ class checkOwner {
     req.foundArticle = response.dataValues;
     return next();
   }
+
+  /**
+   * @author Audace Uhiriwe
+   * @param {req} - request
+   * @param {res} - response
+   */
+  static async profileOwner(req, res, next) {
+    const { id, username } = req.user;
+    const userFound = await User.findOne({ where: { id, username } });
```

Comment:
How does this verify profile ownership? It is just repeating what token verification does. You should remove it.